plugins/cilium-cni/chaining/generic-veth/generic-veth.go

```diff
@@ -87,16 +87,13 @@ func (f *GenericVethChainer) Add(ctx context.Context, pluginCtx chainingapi.Plug
 			}
 
 			addrs, err := netlink.AddrList(link, netlink.FAMILY_V4)
-			if err != nil {
-				return fmt.Errorf("unable to list addresses for link %s: %s", link.Attrs().Name, err)
-			}
-
-			if len(addrs) < 1 {
-				return fmt.Errorf("no address configured inside container")
```

Comment:
while the current behaviour didn't take care of ipv6, it actually follow idea of throwing error if no address configured inside container i.e. fail fast if possible.

We can follow the same idea here i.e. if both addrs and addrsv6 are empty, just throw same error. The changes in this PR is fine as I know that you have a check for vethIP and vethIPv6 below.